tests/sles4sap/sap_deployment_automation_framework/deploy_hanasr.pm

```diff
@@ -73,20 +73,6 @@ sub run {
     # Some playbooks use azure cli
     az_login();
 
-    for my $playbook_options (@execute_playbooks) {
-        # Package 'fence-agents-azure-arm' is not yet installed by SDAF, therefore a workaround has to be applied
-        if ($playbook_options->{playbook_filename} eq 'playbook_04_00_00_db_install.yaml') {
-            record_soft_failure("bsc#1226671 - New package 'fence-agents-azure=arm' has to be installed to prevent HA setup failure");
-            my @cmd = ('ansible', 'QES_DB',
-                "--private-key=$sdaf_config_root_dir/sshkey",
-                '--inventory=' . get_required_var('SAP_SID') . '_hosts.yaml',
-                '--module-name=shell',
-                '--args="sudo zypper in -y fence-agents-azure-arm"');
-            assert_script_run(join(' ', @cmd));
-        }
-        sdaf_execute_playbook(%{$playbook_options}, sdaf_config_root_dir => $sdaf_config_root_dir);
-    }
-
```

Comment:
why removed all the playbooks?

Reply:
Sorry... a blunder from my side. Fixed. Will re-run VRs